docs/how to/use-transferbench.rst

```diff
@@ -1,32 +1,35 @@
 .. meta::
-  :description: TransferBench documentation 
+  :description: TransferBench documentation
   :keywords: TransferBench, API, ROCm, documentation, HIP
```

Comment:
This looks like a repeat of other topics in here. 
Add relevant keywords.

Reply:
Done